plugins/provisioners/chef/config/base_runner.rb

```diff
@@ -81,7 +81,7 @@ def finalize!
           @https_proxy_pass   = nil if @https_proxy_pass == UNSET_VALUE
           @no_proxy           = nil if @no_proxy == UNSET_VALUE
           @node_name          = nil if @node_name == UNSET_VALUE
-          @provisioning_path  = nil if @provisioning_path == UNSET_VALUE
+          @provisioning_path = "/tmp/vagrant-chef" if @provisioning_path == UNSET_VALUE
```

Comment:
Yeah I don't line these long ones up because they'll push everything over 80 chars. A battle of two evils.